test/numerical/TestConv.cpp

```diff
@@ -90,38 +90,39 @@ bool isOMConvTheSameAsNaiveImplFor(const int N, const int C, const int H,
   OwningModuleRef moduleRef(module);
 
   compileModule(moduleRef, ctx, SHARED_LIB_BASE, EmitLib);
-  onnx_mlir::ExecutionSession sess(
-      SHARED_LIB_BASE + ".so", "_dyn_entry_point_main_graph");
+  onnx_mlir::ExecutionSession sess(SHARED_LIB_BASE + ".so", "run_main_graph");
 
-  std::vector<unique_ptr<RtMemRef>> inputs;
-  auto xRmr = unique_ptr<RtMemRef>(getRndRealRmr<float>({N, C, H, W}));
+  std::vector<unique_ptr<RtMemRef, decltype(&rmrDestroy)>> inputs;
```

Comment:
Where is the C++ interface to RMR defined? I saw the C version; is there a "manual" for the C++ version too?

Reply:
This patch concerns the C99 API (important in and of itself, and more significantly building block for JNI). Will add C++ runtime API in another patch shortly.